File: src/test/java/com/bmo/student/StudentTest.java
package com.bmo.student;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    void highScoreTest(){
        Student student = new Student("Bmo",80,60);
        Assertions.assertEquals(80, student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("Bmo",80,60);
        Assertions.assertEquals((80+60)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/bmo/student/StudentTest.kt
package com.bmo.student.kotlin

import com.bmo.kotlin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {

    @Test
    fun highestTest() {
        val student = Student("bmo",80,60)
        Assertions.assertEquals(80, student.highest())
    }

    @Test
    fun averageTest() {
        val student = Student("bmo",80,60)
        Assertions.assertEquals((60+80)/2, student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = Student("bmo",80,60)
        Assertions.assertEquals('C', student.grading())
    }
}